lib/kubernetes-deploy/resource_watcher.rb

```diff
@@ -41,14 +41,23 @@ def run(delay_sync: 3.seconds, reminder_interval: 30.seconds)
     def report_what_just_happened(resources, watch_time)
       new_successes, new_failures = resources.partition(&:deploy_succeeded?)
       new_failures.each do |resource|
+        tags = %W(namespace:#{@namespace} sha:#{ENV['REVISION']} resource:#{resource.id})
         if resource.deploy_failed?
+          ::StatsD.increment('deploy.resource.status', 1, tags: tags << "status:failed")
```

Comment:
I'd say changing the name back makes sense. I don't think having that data is confusing considering the tools name, usage pattern etc.